frontend/public/components/graphs/index.jsx

```diff
@@ -14,39 +15,16 @@ export const Line = props => <AsyncComponent loader={() => import('./graph-loade
 export const Scalar = props => <AsyncComponent loader={() => import('./graph-loader').then(c => c.Scalar)} {...props} />;
 export const Donut = props => <AsyncComponent loader={() => import('./graph-loader').then(c => c.Donut)} {...props} />;
 
-const canAccessPrometheus = (openshiftFlag, prometheusFlag, canListNS) => {
-  if (flagPending(prometheusFlag) || flagPending(openshiftFlag)) {
-    // Wait for feature detection to complete before showing graphs so we don't show them, then hide them.
-    return false;
-  }
-
-  if (!prometheusFlag) {
-    return false;
-  }
-
-  if (!window.SERVER_FLAGS.prometheusBaseURL) {
-    // Proxy has not been set up.
-    return false;
-  }
-
-  if (!openshiftFlag) {
-    return true;
-  }
-
-  // In OpenShift, the user must be able to list namespaces to query Prometheus.
-  return canListNS;
-};
+const canAccessPrometheus = (prometheusFlag) => prometheusFlag && !!prometheusBasePath && !!prometheusTenancyBasePath;
```

Comment:
Just to confirm, this will prevent all `requirePrometheus()` wrapped components from rendering unless ***both*** `prometheusBasePath` and `prometheusTenancyBasePath` are set. Is that what we want?

Reply:
Yeah, I was trying to keep the client logic simple. Currently, the console server will sets both values together, so if one is set the other will always be, too. We assume the RBAC proxy will always be there in OpenShift 4.0. Let me know if that's not the case.